test/integration/framework.go

```diff
@@ -61,7 +64,9 @@ func withConfigGetFreshApiserverAndClient(
 	t *testing.T,
 	serverConfig *TestServerConfig,
 ) (servicecatalogclient.Interface,
+    *server.ServiceCatalogServerOptions,
 	*restclient.Config,
+	clientv3.KV,
```

Comment:
since these are only used in one test, we should make this a new function and have the old function pass through calls to this new one. I wouldn't want people messing with the etcd client in any other case.